src/js/vendor/Recollect.js

```diff
@@ -0,0 +1,63 @@
+import React, {Component} from 'react';
+import Script from 'react-load-script'
+
+class Recollect extends Component {
+
+  getRecollectWidgetName() {
+
+    let name;
+
+    switch(this.props.type) {
+      case 'Collection Schedule Lookup':
+        name = "calendar";
+        break;
+      case 'What do I do with...':
+      default:
+        name = "wizard";
+        break;
+    }
+
+    return name;
+  }
+
+  handleScriptLoad() {
+
+    const name = this.getRecollectWidgetName();
+
+    let script = document.createElement('script');
```

Comment:
Could you maybe add some comments here to describe what happens here and why it's kind janky?

Reply:
yep